examples/stm32/f0/stm32f072-discovery/dac-timer/README

```diff
@@ -0,0 +1,6 @@
+------------------------------------------------------------------------------
+README
+------------------------------------------------------------------------------
+
+DAC with timer to generate triangular waveform
```

Comment:
Either this file, or the code itself, or both should be expanded with the frequency of the waveform, and how that was done.  These are meant to be useful examples, with useufl documentation.